criu/util.c

```diff
@@ -952,6 +952,88 @@ FILE *fopenat(int dirfd, char *path, char *cflags)
 	return fdopen(tmp, cflags);
 }
 
+int fchown_nopriv(int fd, uid_t new_uid, gid_t new_gid)
```

Comment:
imho, this nopriv postfix is misleading. I would suggest to use the cr prefix for all this wrappers (cr_fchodn, cr_fchperm, etc).